servers/quarkus-secrets/src/main/java/org/projectnessie/server/catalog/secrets/AzureSecretsManagerBuilder.java

```diff
@@ -32,10 +32,8 @@ public class AzureSecretsManagerBuilder implements SecretsManagerBuilder {
 
   @Override
   public SecretsManager buildManager() {
-    String prefix = secretsConfig.path();
-    if (!prefix.isEmpty()) {
-      prefix += ".";
-    }
+    String prefix =
+        secretsConfig.path().map(String::trim).map(p -> p.endsWith(".") ? p : p + ".").orElse("");
```

Comment:
nit: move this logic to the config class for sharing?

Reply:
Meh - not for a single char ;)